core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -238,6 +243,52 @@ public void testInlineScript2() {
         }
     }
 
+    public void testInlineScript3() {
+        SearchResponse response = client()
+            .prepareSearch("idx")
+            .addAggregation(
+                dateRange("range")
+                    .field(FIELD_5_NAME)
+                    .addRange("now-1d", "now")
```

Comment:
I thought the idea was to randomize stuff. At least there are several randomized vars in `BucketScriptIT`. I'll fix to a date, but do mind explaining to me when random vars should be used?